client/src/data/constants.tsx

```diff
@@ -2,6 +2,7 @@ import {LngLatBoundsLike} from 'maplibre-gl';
 import {isMobile as isMobileReactDeviceDetect} from 'react-device-detect';
 
 // URLS
+export const DOWNLOAD_ZIP_URL = 'https://justice40-data.s3.amazonaws.com/Score/usa.zip';
```

Comment:
As of just now, you can use this URL to get the real thing: `https://justice40-data.s3.amazonaws.com/data-pipeline/data/score/downloadable/Screening+Tool+Data.zip`